chrome/content/zotero/xpcom/translation/translate_item.js

```diff
@@ -896,4 +899,4 @@ Zotero.Translate.ItemGetter.prototype = {
 		return obj;
 	}
 }
-Zotero.Translate.ItemGetter.prototype.__defineGetter__("numItemsRemaining", function() { return this._itemsLeft.length });
\ No newline at end of file
+Zotero.Translate.ItemGetter.prototype.__defineGetter__("numItemsRemaining", function() { return this._itemsLeft.length });
```

Comment:
That's for unstaged changes, so you'd have had to do a `git reset HEAD^` first, remove the line, commit again, and force push. Anyway, not a big deal — thanks.